sourcecode/apis/contentauthor/Dockerfile

```diff
@@ -3,7 +3,6 @@ FROM php:7.4-fpm-alpine AS php_base
 WORKDIR /app
 
 COPY docker/php.ini $PHP_INI_DIR/conf.d/99-custom.ini
-COPY docker/phpfpm-pool-www.conf /usr/local/etc/php-fpm.d/zz-docker.conf
```

Comment:
With exception of the `listen` directive, all the config in this file duplicated the defaults.